packages/lib/src/authority/ed25519/token.ts

```
@@ -170,6 +171,8 @@ export class Ed25519Authority
     subAccount: SolPublicKeyData;
     roleId: number;
     amount: bigint;
+    allowBelowRentExempt?: boolean;
```

This field should just be used for validation. For example. We could return an error if amount > (balance - rent) and allowBelowRentExempt is false.